protocol/v2/ssv/runner/proposer.go

```diff
@@ -390,3 +412,43 @@ func (r *ProposerRunner) GetRoot() ([32]byte, error) {
 	ret := sha256.Sum256(marshaledRoot)
 	return ret, nil
 }
+
+type blockSummary struct {
+	Hash    phase0.Hash32
+	Blinded bool
+	Version spec.DataVersion
+}
+
+func summarizeBlock(block any) (summary blockSummary, err error) {
+	if block == nil {
```

Comment:
@moshe-blox maybe better to take it out to a separate file?

Reply:
its a small function used in this file for logging info about the proposal, i added some comments on it now

Reply:
also, spec-alignment script automatically skips it (bc there's no such func in spec)

Reply:
ok...